functions/inputs/from_json.go

```diff
@@ -0,0 +1,193 @@
+package inputs
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/influxql"
+	"github.com/influxdata/flux/plan"
+	"github.com/influxdata/flux/semantic"
+	"github.com/pkg/errors"
+)
+
+const FromJSONKind = "fromJSON"
```

Comment:
I'd be concerned that `fromJSON` will be perceived as a general tool of some sort.   namespacing would be the best solution to this problem, but I wonder if we should have this be   `fromInfluxJSON` for now to indicate a fixed JSON structure.

Reply:
I was wondering that myself. I'll change it.